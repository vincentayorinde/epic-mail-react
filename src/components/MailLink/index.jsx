import React from 'react';

const MailLink = ({id, date, sender, title, messageData, onClick, classes}) => (
      <>
      <div onClick={onClick} id={id} className="view record"><span>{date}</span><br />
        <span className="view" id={id}> <h3><i className={classes}></i> {sender}</h3> </span>
        <h4 className="view" id={id}>{title}</h4>
        <p className="view" id={id}>{messageData}</p>
      </div>
      </>
)

export default MailLink;