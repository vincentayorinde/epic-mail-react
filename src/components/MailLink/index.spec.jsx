
import React from 'react';
import { shallow } from 'enzyme';
import MailLink from './index';

describe('MailLink Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<MailLink />);
    expect(component.find('.record').length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
