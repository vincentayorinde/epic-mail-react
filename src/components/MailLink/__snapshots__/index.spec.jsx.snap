// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailLink Component Should render without errors 1`] = `
<Fragment>
  <div
    className="view record"
  >
    <span />
    <br />
    <span
      className="view"
    >
       
      <h3>
        <i />
         
      </h3>
       
    </span>
    <h4
      className="view"
    />
    <p
      className="view"
    />
  </div>
</Fragment>
`;
